components/inputs/input-text.js

```diff
@@ -77,6 +78,10 @@ class InputText extends FormElementMixin(RtlMixin(LitElement)) {
 			 * Imposes a lower character limit
 			 */
 			minlength: { type: Number },
+			/**
+			 * Additional information communicated in the aria-describedby on the input
+			 */
+			offscreenDescription: { type: String, reflect: true, attribute: 'offscreen-description' },
```

Comment:
What do you think about just calling this `description`? We use that in buttons as "for accessibility when text on button does not provide enough context" which is kind of what this is. How input chooses to render that (offscreen linked with `aria-describedby` or otherwise) seems like an internal implementation detail.

Reply:
Yup that makes sense to me.